jipdate.py

```diff
@@ -337,6 +341,16 @@ def create_default_config(config_file):
 # Extra comments added to each Jira issue (multiline is OK)
 comments:
         - "No updates since last week."
+
+# Header of the file (multiline is OK). It will be followed by JIRA_USERNAME
+header:
```

Comment:
Another comment, if you write the section below like in my patch, then the config.yml will be a bit cleaner (or easier to read).
```patch
diff --git a/jipdate.py b/jipdate.py
index f0d6a9b..8d8c968 100755
--- a/jipdate.py
+++ b/jipdate.py
@@ -344,14 +344,12 @@ comments:
 
 # Header of the file (multiline is OK). It will be followed by JIRA_USERNAME
 header:
-        - "Hi,
+        - |
+          Hi,
 
-This is the status update from me for the last week.
-
-Cheers!
-"
+          This is the status update from me for the last week.
 
-"""
+          Cheers! """
     with open(config_file, 'w') as f:
         f.write(yml_cfg)
 ```
Source: http://stackoverflow.com/questions/3790454/in-yaml-how-do-i-break-a-string-over-multiple-lines